zeebe/engine/src/main/java/io/camunda/zeebe/engine/state/routing/DbRoutingState.java

```diff
@@ -126,6 +126,14 @@ public boolean activatePartition(final int partitionId) {
     }
   }
 
+  @Override
+  public void setMessageCorrelation(final MessageCorrelation messageCorrelation) {
```

Comment:
:question: Since this is a new code, I was wondering is it not updated after scaling? How does message routing works correctly then?

Reply:
After scaling, `messageCorrelation` remains untouched until message relocation is implemented.